csss-site/src/csss/static/csss_static/mm_hackathon/main.css

```diff
@@ -0,0 +1,512 @@
+html {
```

Comment:
this should be under the location `csss-site/src/events/static/events_static/mm2020_main.css`